lib/client/order.rb

```diff
@@ -75,7 +109,14 @@ def get_transactions(order_id, params = {})
           response = client.get(uri({ api_version: 'v3', order_id: order_id }, 'transactions'), params)
           return response.body['data']
         rescue Faraday::Error => e
-          raise BigcommerceApiError.new('get order transactions', { order_id: order_id }, e)
+          raise BigcommerceApiError.new(
+            e.response[:status],
+            'get order transactions',
+            "Failed to get order transactions #{order_id}",
+            order_id,
+            { order_id: order_id },
```

Comment:
Drop the `order_id` in the data object, and add the BC errors.

Reply:
@mrogers-5s resolved